DockDoor/Utilities/DockObserver.swift

```diff
@@ -539,4 +549,20 @@ final class DockObserver {
             }
         }
     }
+
+    private func handleCmdRightClickQuit(app: NSRunningApplication, event: CGEvent) {
+        Task { @MainActor in
+            // Check if Option key is also pressed for force quit
+            let shouldForceQuit = event.flags.contains(.maskAlternate)
+
+            if shouldForceQuit {
+                app.forceTerminate()
+            } else {
+                app.terminate()
+            }
+
+            // Purge the app cache
```

Comment:
This is not required as it will be called by the termination observer - thanks for the PR!